daemon/cmd/daemon_main.go

```diff
@@ -1013,8 +1019,8 @@ func initEnv(cmd *cobra.Command) {
 	bpf.CheckOrMountFS(option.Config.BPFRoot, k8s.IsEnabled())
 	cgroups.CheckOrMountCgrpFS(option.Config.CGroupRoot)
 
-	option.Config.Opts.SetBool(option.Debug, false)
-	option.Config.Opts.SetBool(option.DebugLB, false)
+	option.Config.Opts.SetBool(option.Debug, debugDatapath)
```

Comment:
Also, perhaps it would be a good idea to add a note in the flag description of the `debug` flag that it no longer enables debug messages in the datapath and it should users should use `debug-verbose=datapath` instead.

Reply:
^ Probably needs backport right?

Reply:
@joestringer this PR is already marked for backport, was that your concern?